storage/clickhousespanstore/tablename_test.go

```diff
@@ -0,0 +1,27 @@
+package clickhousespanstore
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTableName_AddDbName(t *testing.T) {
+	assert.Equal(t, TableName("database_name.table_name_local"), TableName("table_name_local").AddDbName("database_name"))
+}
+
+func TestTableName_ToGlobal(t *testing.T) {
```

Comment:
`ToGlobal` has maybe too ambiguous definition. Maybe it's better to define `ToLocal` and always add `_local` suffix.

Reply:
Done in #67